spec/factories/application_choice.rb

```diff
@@ -3,6 +3,10 @@
     course_option
     application_form
 
+    after(:build) do |application_choice, _evaluator|
+      application_choice.current_option = application_choice.course_option
```

Comment:
This is lying to us until we've done the migration, right?

Reply:
The factory will set `current_course_option_id` only if it isn't set, in which case it'll use the original `course_option`.